lib/phoenix/endpoint.ex

```diff
@@ -768,6 +768,8 @@ defmodule Phoenix.Endpoint do
 
         * `:peer_data` - the result of `Plug.Conn.get_peer_data/1`
 
+        * `:trace_context_headers` - a list of all trace context headers
```

Comment:
Should include examples in the docs.

Reply:
I figured we'd have examples in https://github.com/opentelemetry-beam/opentelemetry_phoenix and the main lib. Maybe one in the telemetry guide in this repo, too?

Reply:
A link to the W3C spec as already mentioned in the PR message would be nice here too.